configure.sh

```diff
@@ -276,6 +277,8 @@ CONFIG_HOST=${CONFIG_HOST}
 MAKECONF_CC=${CC}
 MAKECONF_LD=${LD}
 CONFIG_SPT_NO_PIE=${CONFIG_SPT_NO_PIE}
+SSP_GUARD=${SSP_GUARD}
+SSP_FAIL=${SSP_FAIL}
```

Comment:
Rather than passing `SSP_GUARD` and `SSP_FAIL` all the way up from `configure.sh` into the bindings Makefile, could you instead please use an `ifeq ($(CONFIG_HOST), OpenBSD) ... else ... endif` block in `bindings/GNUmakefile`, just before the `define LINK.bindings`?